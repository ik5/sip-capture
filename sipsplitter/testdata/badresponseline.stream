SIP/2.0 OK
Content-Length: 0

