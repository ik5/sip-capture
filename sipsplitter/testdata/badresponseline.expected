{
  "Original": "SIP/2.0 OK\r\nContent-Length: 0\r\n\r\n",
  "Discarded": 0,
  "NoStartLine": 1,
  "StartLine": 0,
  "NoHeaders": 0,
  "Header": 0,
  "NoBody": 0,
  "Body": 0,
  "Complete": 0,
  "Discards": [],
  "Starts": [],
  "Headers": [],
  "Bodies": [],
  "Messages": []
}